handler/src/main/java/io/netty/handler/ssl/OpenSslClientContext.java

```diff
@@ -54,7 +58,21 @@ public OpenSslClientContext() throws SSLException {
      */
     @Deprecated
     public OpenSslClientContext(File certChainFile) throws SSLException {
-        this(certChainFile, null);
+        this(certChainFile, null, KeyStore.getDefaultType());
+    }
+
+    /**
+     * Creates a new instance.
+     *
+     * @param trustManagerFactory the {@link TrustManagerFactory} that provides the {@link TrustManager}s
+     *                            that verifies the certificates sent from servers.
+     *                            {@code null} to use the default.
+     * @param keyStore the KeyStore this context should use
+     * @deprecated use {@link SslContextBuilder}
+     */
+    @Deprecated
+    public OpenSslClientContext(TrustManagerFactory trustManagerFactory, String keyStore) throws SSLException {
```

Comment:
It does not make sense to expose a protected method and mark it as @Deprecated from the beginning. Maybe you should remove the annotation and make it package-private ?